sync/src/relayer/transactions_process.rs

```diff
@@ -0,0 +1,176 @@
+use crate::relayer::compact_block::RelayTransactions;
+use crate::relayer::Relayer;
+use ckb_logger::debug_target;
+use ckb_network::{CKBProtocolContext, PeerIndex};
+use ckb_protocol::RelayTransactions as FbsRelayTransactions;
+use ckb_store::ChainStore;
+use failure::Error as FailureError;
+use fnv::FnvHashSet;
+use futures::{self, future::FutureResult, lazy};
+use std::convert::TryInto;
+use std::sync::Arc;
+use std::time::Duration;
+
+const DEFAULT_BAN_TIME: Duration = Duration::from_secs(3600 * 24 * 3);
+
+pub struct TransactionsProcess<'a, CS> {
+    message: &'a FbsRelayTransactions<'a>,
+    relayer: &'a Relayer<CS>,
+    nc: Arc<dyn CKBProtocolContext + Sync>,
+    peer: PeerIndex,
+}
+
+impl<'a, CS: ChainStore + Sync + 'static> TransactionsProcess<'a, CS> {
+    pub fn new(
+        message: &'a FbsRelayTransactions,
+        relayer: &'a Relayer<CS>,
+        nc: Arc<CKBProtocolContext + Sync>,
+        peer: PeerIndex,
+    ) -> Self {
+        TransactionsProcess {
+            message,
+            relayer,
+            nc,
+            peer,
+        }
+    }
+
+    pub fn execute(self) -> Result<(), FailureError> {
+        let relay_txes: RelayTransactions = (*self.message).try_into()?;
```

Comment:
Please use `txs` as `transaction`'s plural form abbreviation:
```suggestion
        let relay_txs: RelayTransactions = (*self.message).try_into()?;
```

Reply:
thanks.